src/cli/index.js

```diff
@@ -9,8 +10,50 @@ import log from './log';
 import remotesWizard from './remotes-wizard';
 import createTestCafe from '../';
 
-function exit (code) {
+
+const TERMINATION_TYPES = {
+    sigint:   'sigint',
+    shutdown: 'shutdown'
+};
```

Comment:
On Windows platform is possible to terminate cli by pressing `CTRL+BREAK`.
I propose to support `SIGBREAK` signal.
See https://nodejs.org/api/process.html#process_signal_events